src/microbe_stage/editor/MicrobeEditorGUI.cs

```diff
@@ -1000,6 +1005,21 @@ public void UpdateMicrobePartSelections()
         }
     }
 
+    /// <summary>
+    ///   Updates the visibility of the move cancel button.
```

Comment:
This should be reworded to be more general as the cancel button may be used for other actions in the future.